.pylintrc

```diff
@@ -0,0 +1,419 @@
+[MASTER]
```

Comment:
Do we really need this huge file. Aren't there some sane defaults we can base on?